scripts/generate_errors.pl

```diff
@@ -46,6 +45,44 @@
 
 $/ = $line_separator;
 
+=comment
+open(ERROR_H, "<", $error_h) or die "Opening error.h";
+my %sets;
+my $section == 0;
```

Comment:
this should be `=`

Reply:
Or maybe given the size of the patch, if we revive this, we'll rewrite the whole script in python.